### PR TITLE
Version 3.8.0 release notes

### PR DESCRIPTION
**Pull request by jymaps**

### CODE REVIEW

File: collections/_cms/changelog.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ date: 2019-10-07
 published: true
 ---
 
+## January 2020 - v3.8.0
+
+### What’s New:
+
+* **CAD file upload:** Ability to upload a CAD file (.dwg, .dxf) per floor in the CMS. You can find a guide in the documentation https://mapsindoors.github.io/cms/#buildings.
```
Comment (marksdk) [inline collections/_cms/changelog.md]:
```suggestion
* **CAD file upload:** Ability to create new buildings and floors and upload CAD files (.dwg, .dxf) for the floors in the CMS. You can find [a guide in the documentation](https://mapsindoors.github.io/cms/#buildings).
```

File: collections/_cms/changelog.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ date: 2019-10-07
...
+* **Icon Manager:** Added the option to “skip” uploading icons larger than 128x64px.
```
Comment (marksdk) [inline collections/_cms/changelog.md]:
```suggestion
* **Icon Manager:** Added the option to "skip" uploading icons larger than 128x64px.
```

File: collections/_cms/changelog.md [hunk 3]
```diff
@@ -4,6 +4,20 @@ date: 2019-10-07
...
+  * When deleting a used icon from the Icon Manager, it is also removed from the map.
```
Comment (marksdk) [inline collections/_cms/changelog.md]:
```suggestion
  * When deleting an icon in the Icon Manager that is in use on the map, it is also removed from the map.
```

File: collections/_cms/changelog.md [hunk 4]
```diff
@@ -4,6 +4,20 @@ date: 2019-10-07
...
+  
+### Bug Fixes:
+
+* **Icon Manager:** shows a more descriptive error message when an image fails.
```
Comment (marksdk) [inline collections/_cms/changelog.md]:
```suggestion
* **Icon Manager:** Shows a more descriptive error message when an image fails.
```

File: collections/_cms/changelog.md [hunk 5]
```diff
@@ -4,6 +4,20 @@ date: 2019-10-07
...
+* **Locations:** Google Street View network now available inside The White House demo solution.
+* **Route Access:** Fixed issue where eternal doors showed up as elevator doors.
```
Comment (marksdk) [inline collections/_cms/changelog.md]:
```suggestion
* **Route Access:** Fixed issue where external doors showed up as elevator doors.
```

I really like the idea of _eternal doors_, but I think you meant _external_ 😉

Comment (jymaps) [reply collections/_cms/changelog.md]: 😄

**Comment by marksdk** [comment]
@jymaps I merged the linted documents into this branch. Will you check that the Changelog is still correct before merging this in?

**Pull request merged**